### PR TITLE
Add wandb_entity to wandb options, update example configs, update README

### PR DESCRIPTION
**Pull request by morganmcg1**
- Update wandb_entity and add wandb descriptions
- add wandb to config section

### CODE REVIEW

**Review by seven-dev — APPROVED** [review]
Maybe the configs could be checked against a validator, just to keep track of the current and deprecated ones...
I'm kind of a newbie here but it seems good to me.

**Review by winglian — APPROVED** [review]
Thanks @morganmcg1 !

File: README.md [hunk 1]
```diff
@@ -401,11 +401,12 @@ lora_out_dir:
...
 # wandb configuration if you're using it
-wandb_mode:
-wandb_project:
-wandb_watch:
-wandb_run_id:
-wandb_log_model: # 'checkpoint'
+wandb_mode: # "offline" to save run metadata locally and not sync to the server, "disabled" to turn off wandb
+wandb_project: # your wandb project name
+wandb_entity: # a wandb Team name if using a Team
+wandb_watch:
```
Comment (winglian) [inline README.md]:
```suggestion
wandb_watch:
```

File: README.md [hunk 2]
```diff
@@ -549,6 +550,18 @@ fsdp_config:
...
+##### Weights & Biases Logging
+
+- wandb options
+```yaml
+wandb_mode: 
+wandb_project: 
+wandb_entity: 
+wandb_watch: 
+wandb_run_id: 
+wandb_log_model:
```
Comment (winglian) [inline README.md]:
```suggestion
wandb_mode:
wandb_project:
wandb_entity:
wandb_watch:
wandb_run_id:
wandb_log_model:
```

**Review by winglian — APPROVED** [review]
just need to remove the trailing whitespace for the pre-commit linter

**Pull request merged**